Drop LSM file

pinotree opened the pull request:
The LSM format is a very old format for software, and it is dead for many years already. Since keeping the .lsm up-to-date is basically not much of use, simply drop it altogether.

dreibh commented:
The LSM format is old, but I am not sure whether some packagers still use it for something. Since this file is updated by a small script (using the information from debian/control), it is easy to still keep and maintain it for a while.

pinotree commented:
> Since this file is updated by a small script (using the information from debian/control),

It doesn't seem to be the case:
- the file currently points to 1.3.1, whereas the latest version at this time is 1.3.5
- the previous version bump was from 1.2.0 to 1.3.1, skipping 1.3.0
- I don't see anything in repository referencing that file

> I am not sure whether some packagers still use it for something.

Considering that you can find literally a handful of LSM files (no more than 10, maybe 20 if you are lucky) in the sources of a modern Linux distro, I don't see what packagers would do with a dead format.

pinotree committed:
Drop LSM file

The LSM format is a very old format for software, and it is dead for
many years already. Since keeping the .lsm up-to-date is basically not
much of use, simply drop it altogether.

dreibh commented:
The LSM file is gone now -> case solved.
